.php_cs.dist

```diff
@@ -28,7 +28,7 @@ return PhpCsFixer\Config::create()
         'no_null_property_initialization' => true,
         'no_short_echo_tag' => true,
         'no_unreachable_default_argument_value' => true,
-        'no_useless_else' => true,
+        'no_useless_else' => ['convert_elseif_to_if' => true],
```

Comment:
not sure about the option... imo, it's just a next case we didn't spot before, shall be fixed always.
WDYT @julienfalque @SpacePossum